src/python/pants/core/goals/fmt.py

```diff
@@ -38,6 +41,14 @@ class FmtResult(EngineAwareReturnType):
     stderr: str
     formatter_name: str
 
+    def __post_init__(self):
+        log = f"Output from {self.formatter_name}"
+        if self.stdout:
+            log += f"\n{self.stdout}"
+        if self.stderr:
+            log += f"\n{self.stderr}"
+        logger.debug(log)
```

Comment:
Nit, maybe worth a comment explaining why we do this. It looks out of place to a lay reader, I think, and doesn't capture that it was very intentional to add this.